README.md

```diff
@@ -2,6 +2,36 @@
 
 This repo implements the CSI spec v1.1.0 for [GIG.tech's OpenvCloud](https://gig.tech)
 
+## Ansible role CSI-Driver
+
+OVC CSI driver can be installed with Ansible role [csi-driver](roles/scr-driver)
+
+Role variables:
+
+``` yaml
+# required
+server_url: "<G8 url>"
+
+cluster_url: "https://your-g8.gig.tech"
+account: "your-account-name"
+client_jwt: "ItsyoOnline-JWT-token"
+persistent_volume_size: "Size of persistent storage"
+```
+
+Example of the configuration file setting credentials and required config see in [config.env.example](config.env.example).
```

Comment:
Add that people need to copy `config.env.example` to `config.env` and add their data in that file, noting that `config.env` is in gitignore and presents a push of their credentials to the repo.

Reply:
why would a user push anything to the repo?

Reply:
The user could be a developer, that's why we add the `config.env.example` and `config.env.` in gitignore in the first place, directly editing in `config.env.example` defeats this whole purpose